js/gui.js

```diff
@@ -103,6 +103,11 @@ var keyHandlers = {
     },
 };
 
+// Reset button
+function reset() {
+    location.reload();
```

Comment:
that is a good start but the game is also able to do that without reloading as we do not use global state. We can change that later/create an issue.